activemodel/lib/active_model/type/helpers/numeric.rb

```diff
@@ -8,14 +8,21 @@ def serialize(value)
           cast(value)
         end
 
+        def serialize2(value)
+          cast(value)
+        end
+
         def cast(value)
-          value = \
+          value = if value <=> 0
```

Comment:
We should probably add a comment here about why we're doing this.  This is a huge "hack" to check to see whether or not `value` is numeric.  Non-numbers will return `nil` for `<=>`, and it avoid us need to do an `is_a?` check.  We found using the spaceship operator was faster than checking types.

Reply:
I'm curious why not use `value.is_a?(Numeric)`. `<=>` is faster alternative of `value.is_a?(Numeric)`?